src/blob_gc_job_test.cc

```diff
@@ -196,7 +211,7 @@ class BlobGCJobTest : public testing::Test {
     DestroyDB();
   }
 
-  void TestRunGC() {
+  void TestDoGC(bool run_gc) {
```

Comment:
Please check the file_size in this function

Reply:
file size had been checked in line 258